app/models/content_item.rb

```diff
@@ -1,9 +1,11 @@
 require "ostruct"
 
 class ContentItem
+  include Withdrawable
```

Comment:
Is Withdrawable definitely something that all ContentItems (or, let's say, the overwhelming majority of them) are going to use?